src/main/java/seedu/exercise/model/Model.java

```diff
@@ -175,16 +182,18 @@
      */
     void addSchedule(Schedule schedule);
 
-    /** Returns the data in the regime book */
-    ReadOnlyScheduleBook getAllScheduleData();
+    /**
+     * Returns the data in the regime book
+     */
+    ReadOnlyResourceBook<Schedule> getAllScheduleData();
 
     /**
      * Deletes a Schedule and adds it to {@code ExerciseBook} for tracking.
-     *
+     * <p>
      * If the schedule has some exercises that are duplicates exercises as
-     * specified by {@link Exercise#isSameExercise}, that exercise will
+     * specified by {@link Exercise#isSameResource}, that exercise will
      * be ignored and not be added into the exercise tracker.
-     *
+     * <p>
```

Comment:
no closing \<p> tags here. Add in the \</p>

Reply:
done